### PR TITLE
tools to create the translation keys for tooltips

### PR DESCRIPTION
**Pull request by rmichaelis**
@petzlux Could you please review ?

### CODE REVIEW

**Comment by petzlux** [comment]
okay looking at it

**Comment by rmichaelis** [comment]
I just would like to be sure, that I will not delete all the translations in transifex.

**Comment by petzlux** [comment]
Are these error messages expected ? 
````
Error with the layer:  1291 using query : 
http://10.112.176.27:6080/arcgis/rest/services/Geoportail/Geoportail/MapServer/116/query?
&outFields=%2A&text=&outSR=&spatialRel=esriSpatialRelIntersects&returnGeometry=true&inSR=2169
&f=pjson&geometry=96958.90059551848%2C61965.61097091329%2C97454.77280739773%2C62463.
21618929457&objectIds=&geometryType=esriGeometryEnvelope&where= response: {'error': {'message': 
'Invalid or missing input parameters.', 'code': 400, 'details': []}}
````

**Comment by rmichaelis** [comment]
Yes the remote webservice has no attribute to publish and returns an error.
In this case, I just log the error.

**Comment by petzlux** [comment]
@rmichaelis can you please review my changes, especially if the build now works as expected and integrates any updates to the tooltip translations to geoportailv3.json on the client

File: geoportailv3/locale/de/LC_MESSAGES/geoportailv3-client.po [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#
```
Comment (rmichaelis) [inline geoportailv3/locale/de/LC_MESSAGES/geoportailv3-client.po]: Do we have to commit the po file ?

**Comment by petzlux** [comment]
@rmichaelis can you check as it didn't create the geoportailv3.json everytime

**Comment by rmichaelis** [comment]
@petzlux I have this error message : 
msguniq update-tooltips -o update-tooltips
msguniq: error while opening "update-tooltips" for reading: No such file or directory
make: *** [update-tooltips] Error 1

**Comment by petzlux** [comment]
@pgiraud ready for review :-)

**Comment by jaykayone** [comment]
transifex hans on bad ssl certificate :-1:

**Comment by rmichaelis** [comment]
@petzlux I still have this error message, do you know where it comes from ?: 
msguniq update-tooltips -o update-tooltips
msguniq: error while opening "update-tooltips" for reading: No such file or directory
make: *** [update-tooltips] Error 1

**Comment by rmichaelis** [comment]
Ok I now understand. 
The make "update-tooltips" target should not be used. I have to use the update-translation target.

**Comment by jaykayone** [comment]
Yes, but the update-translations target does not always regenerate the pot file

**Pull request merged**